_tools/fetch-team-data.js

```diff
@@ -77,7 +77,11 @@ const team = {
                 username: user.login,
                 name: profile.name,
                 website: profile.blog,
-                avatar_url: profile.avatar_url
+                avatar_url: profile.avatar_url,
+                bio: profile.bio,
+                twitter_username: profile.twitter_username,
+                github_username: user.login,
```

Comment:
`github_username` looks redundant, don't we already have GitHub username in `username` field?

Reply:
Yes. It’s redundant for now because I don’t want to go through and change all the references in the current site. In the new site, it’s helpful to have the username source available to make it to clear where the username is used. Once we are fully on the new site we can remove the username field.